launch/soccer.launch.py

```diff
@@ -42,20 +55,39 @@ def generate_launch_description():
     direction_flag = LaunchConfiguration("direction_flag", default="plus")
     use_manual_control = LaunchConfiguration("use_manual_control", default="False")
 
+    # args that can be set from cmd line
+    # https://docs.ros.org/en/foxy/How-To-Guides/Launch-file-different-formats.html
+    # TODO: match above args to this style of declaring args, since this is cleaner
```

Comment:
make one big issue for all the small launch file cleanup TODOs and do it next PR

Reply:
nah I'll just do this in a PR